CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright 2018 VMware, Inc.
+    Copyright 2018-2019 VMware, Inc.
```

Comment:
VMware doesn't do year ranges.  Only the year of the last significant change is listed.

Reply:
Thanks, I'll update the PR.